brainstormforce/ultimate-addons-for-gutenberg
# Resolved issue of get FSE theme content width using global setting for Container.

### Description

- Container content width

### Screenshots

- https://share.getcloudapp.com/YEueyNAx
- https://share.getcloudapp.com/d5ugp8pN

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Switch to Twenty Twenty-Three.
- Theme editor > Page template > global style > Layout > set Content width. Save it.
- Visit and Spectra setting, and you will notice Default Content Width inherited in the input field. Also, notice will be shown - https://share.getcloudapp.com/12u7rByy
- Visit the editor drag and drop the new container you will see the content width set to the global content width.
- Save the changes and visit the front end. you will see it applied properly.
- Make sure it does not hamper the already set content width for the container. 

### Checklist:
